packages/core/lib/js-runtime.js

```diff
@@ -0,0 +1,5 @@
+module.exports = process.env.IS_BROWSER
+  ? 'browserjs'
+  : ((typeof navigator !== 'undefined' && navigator.product === 'ReactNative')
+    ? (typeof Expo !== 'undefined' ? 'expojs' : 'reactnativejs')
+    : 'nodejs')
```

Comment:
The rare 

_T R I P L E_
 _T E R N A R Y_

![](https://media.giphy.com/media/3o7qDVArtXdtl223Ek/giphy.gif)